conf/modules/call_repeat_expansions.config

```diff
@@ -24,11 +24,12 @@ process {
     }
 
     withName: '.*CALL_REPEAT_EXPANSIONS:EXPANSIONHUNTER' {
-        ext.args =  { ("${meta.sex}" == 1) ? '--sex male' : '--sex female' }
+        ext.args =  { ("${meta.sex}" == '1') ? '--sex male' : '--sex female' }
```

Comment:
Schema does expect the value to be an integer, but in meta everything is of string type.. I retrieved the type of the values in our meta and this is what it looks like.. 
`[id:java.lang.String, sample:java.lang.String, lane:java.lang.String, sex:java.lang.String, phenotype:java.lang.String, paternal:java.lang.String, maternal:java.lang.String, case_id:java.lang.String, num_lanes:java.lang.String, read_group:java.lang.String, single_end:java.lang.Boolean]`

Reply:
I see, didn't know that :) 
In that case we should look into whether this needs changing this as well
https://github.com/nf-core/raredisease/blob/2be203dbf3839ac3e740038d1f77e38e17d85493/conf/modules/generate_cytosure_files.config#L38
Worked for my testing previously though
Also it might be worth looking into whether this function needs changing
https://github.com/nf-core/raredisease/blob/2be203dbf3839ac3e740038d1f77e38e17d85493/workflows/raredisease.nf#L745